### PR TITLE
Adding modules for each JER / JES variation. 

### PR DESCRIPTION
**Pull request by hayfasfar**

### CODE REVIEW

File: processors/HNL.py [hunk 1]
```diff
@@ -203,6 +194,65 @@
...
+			)
+		)
+	
+	for systName,jetCollection,metObject in [
+        	("nominal",lambda event: event.selectedJets_nominal,lambda event: event.met_nominal),
+	        ("jerUp",lambda event: event.selectedJets_jerUp,lambda event: event.met_jerUp),
+	        ("jerDown",lambda event: event.selectedJets_jerDown,lambda event: event.met_jerDown),
+	        ("jesTotalUp",lambda event: event.selectedJets_jesTotalUp,lambda event: event.met_jesUp["Total"]),
+	        ("jesTotalDown",lambda event: event.selectedJets_jesTotalDown,lambda event: event.met_jesDown["Total"]),
+        	("unclEnUp",lambda event: event.selectedJets_nominal,lambda event: event.met_unclEnUp),
```
Comment (daiktas) [inline processors/HNL.py]: what is here?

File: processors/HNL.py [hunk 2]
```diff
@@ -203,6 +194,65 @@
...
+            		JetSelection(
+                		inputCollection=collection,
+				leptonCollection=lambda event: event.tightMuon,		
+	                	outputName="selectedJets_"+systName,
+		                storeKinematics=['pt','eta'],
+        			globalOptions=globalOptions
+			)
+	        )
+
+
```
Comment (daiktas) [inline processors/HNL.py]: should also add the lepJet selection here for each variation

File: processors/HNL.py [hunk 3]
```diff
@@ -203,6 +194,65 @@
...
+        			globalOptions=globalOptions
+			)
+	        )
+
+	
+		analyzerChain.append(
+			EventObservables(
+			jetCollection = collection,
+			leptonCollection = lambda event: event.tightMuon[0],
+			outputName="Jet_Muon_"+systName
```
Comment (daiktas) [inline processors/HNL.py]: We should change this name, I put it like that but it's a bit stupid. Maybe "EventObservables_"+systName

File: processors/HNL.py [hunk 4]
```diff
@@ -121,8 +121,9 @@
 
 analyzerChain.append(
```
Comment (daiktas) [inline processors/HNL.py]: This module is now redundant because you have the same one "nominal" in the loop

File: processors/HNL.py (hunk 1 above)
Comment (daiktas) [reply processors/HNL.py]: the unclEnUp

File: processors/HNL.py (hunk 4 above)
Comment (daiktas) [reply processors/HNL.py]: Put every other module that depends on the jets also in the loop, like the LepJet selection

File: processors/HNL.py (hunk 3 above)
Comment (hayfasfar) [reply processors/HNL.py]: this is the output name for the event observables. 
outputName = "Jet_Muon", 
So i would like to keep the same module output +systName.

File: processors/HNL.py (hunk 2 above)
Comment (hayfasfar) [reply processors/HNL.py]: ok  true

File: processors/HNL.py (hunk 3 above)
Comment (daiktas) [reply processors/HNL.py]: I'd suggest changing the default name in EventObservables.py to "EventObservables" too

Comment (hayfasfar) [reply processors/HNL.py]: it was like that before , and it was who changed it to Jet_Muon....

Comment (daiktas) [reply processors/HNL.py]: As I said in my first comment, that was a bit stupid..

Comment (hayfasfar) [reply processors/HNL.py]: i don't really see what is your issue here, kind of NNNNLO detail. But Ok i will call it as you want..

File: processors/HNL.py (hunk 1 above)
Comment (hayfasfar) [reply processors/HNL.py]: it is the uncertainty due to the unclustered jets below 15 GeV there is also the down one

**Pull request merged**